pkg/clusterapi/README.md

```diff
@@ -0,0 +1,42 @@
+# Cluster API
```

Comment:
README is great. You might consider putting it under the top level `/docs/` dir.